Documentation/ceph-cluster-crd.md

```diff
@@ -24,6 +24,13 @@ spec:
     # see the "Cluster Settings" section below for more details on which image of ceph to run
     image: ceph/ceph:v13.2.2-20181023
   dataDirHostPath: /var/lib/rook
+  # set the amount of mons to be started
+  mon:
+    count: 3
+    allowMultiplePerNode: true
```

Comment:
To keep the examples simple, what if we only include the minimal properties that will work, while also making the point for the example? Can we consider removing the following properties and go with the default? 
- `allowMultiplePerNode`, `dashboard`, `network`

I'm struggling with the balance between showing a simple example that can be understood and also showing all the possible properties so they know what they can configure.

Reply:
I would like to keep `allowMultiplePerNode: true` in as the default is `false` making this example not work on one node clusters.
The others should be fine, will remove them.

Reply:
I have just checked the code, `dashboard.enabled: true` is not the default.
I think it would be good to at least have `dashboard.enabled: true` in the first example (without a comment).